README.md

```diff
@@ -270,8 +270,9 @@ Calling this method after the response has ended/closed is a NOOP.
 
 Unless you specify a `Content-Length` header yourself, HTTP/1.1 responses
 will automatically use chunked transfer encoding and send the respective header
-(`Transfer-Encoding: chunked`) automatically. If you know the length of your
-body, you MAY specify it like this instead:
+(`Transfer-Encoding: chunked`) automatically. If `Transfer-Encoding` is already given
+the `Content-Length` will be removed.
```

Comment:
Thanks, makes much more sense like this to me :+1: The consumer never receives a `Transfer-Encoding` (via #137) and thus should have no control over sending one either :+1: